test/contrib/emscripten/conftest.py

```diff
@@ -158,27 +248,110 @@ def run_webworker(self, code: str) -> Any:
 def run_from_server(
     selenium_coverage: Any, testserver_http: PyodideServerInfo
 ) -> Generator[ServerRunnerInfo, None, None]:
-    addr = f"https://{testserver_http.http_host}:{testserver_http.https_port}/pyodide/test.html"
-    selenium_coverage.goto(addr)
-    selenium_coverage.javascript_setup()
-    selenium_coverage.load_pyodide()
-    selenium_coverage.initialize_pyodide()
-    selenium_coverage.save_state()
-    selenium_coverage.restore_state()
-    # install the wheel, which is served at /wheel/*
-    selenium_coverage.run_js(
-        """
-await pyodide.loadPackage('/wheel/dist.whl')
-"""
-    )
-    selenium_coverage._install_coverage()
+    if selenium_coverage.browser != "node":
+        # on node, we don't need to be on the same origin
+        # so we can ignore all this
```

Comment:
There is a way to disable it but it is deep in the firefox profile meaning it would require code in pytest-pyodide or monkeypatching; making it be on the same origin is easier.